pkg/dao/repository.go

```diff
@@ -32,13 +34,31 @@ func DBErrorToApi(e error) error {
 }
 
 func (r repositoryDaoImpl) Create(newRepo api.RepositoryRequest) error {
-	newRepoConfig := models.RepositoryConfiguration{
-		AccountID: *newRepo.AccountID,
-		OrgID:     *newRepo.OrgID,
+	url := ""
+	if newRepo.URL != nil {
+		url = *newRepo.URL
+	}
```

Comment:
`api.createRepository` calls `FillDefaults`, which already checks for `nil` like this and replaces it with `""`

Reply:
was the redundant error check intentional?

Reply:
actually I think it makes sense to error check in both places since these are separate layers